packages/xlucene-evaluator/src/utils.ts

```diff
@@ -169,6 +178,124 @@ export type CreateJoinQueryOptions = {
     arrayJoinBy?: JoinBy;
 };
 
+function isGeoJSONData(input: any): input is GeoShape {
+    return input.coordinates != null
+        && Array.isArray(input.coordinates)
+        && input.type != null;
+}
+
+type JoinGeoShape = GeoShape | ESGeoShape;
+
+const relationList = Object.values(GeoShapeRelation);
+
+export function makeXluceneGeoDistanceQuery(
+    field: string,
+    value: GeoPointInput,
+    fieldParam?: string
+) {
+    const distance = fieldParam ? escapeValue(fieldParam) : '"100m"';
+    const results = parseGeoPoint(value, false);
+    if (!results) return '';
+    const { lat, lon } = results;
+    return `${field}:geoDistance(point:"${lat},${lon}" distance:${distance})`;
+}
+
+export function makeXlucenePolyContainsPoint(field: string, value: GeoPointInput) {
+    const results = parseGeoPoint(value, false);
+    if (!results) return '';
+    const { lat, lon } = results;
+    return `${field}:geoContainsPoint(point:"${lat},${lon}")`;
+}
+
+function coordinateToXlucene(cord: CoordinateTuple) {
+    // tuple is [lon, lat], need to return "lat, lon"
+    return `"${cord[1]}, ${cord[0]}"`;
+}
+
+function wrap(arr: any[]) {
```

Comment:
This function name should be more specific. Also is the function properly stringifying the array?

```suggestion
function makeArrayString(arr: any[]) {
```